src/commands/cluster-slots.json

```diff
@@ -33,6 +33,8 @@
             "type": "array",
             "items": {
                 "type": "array",
+                "minItems": 3,
+                "maxItems": 100000,
```

Comment:
`additionalItems` is replica address. So, there might be zero or many replicas. If we don't mention maxItems, our script will complain about missing `maxItems` field. Any suggestion to handle this case?

Reply:
let's go with UINT_MAX (4294967295)